study.md

```diff
@@ -50,15 +50,17 @@ In your own words, define what the responsibilities of the controller layer are
 in Rails.
 
 ```md
-<!-- your answer here -->
+The controller takes the data from the model and puts it in the view. You only manipulate data in the controllers.
 ```
 
 ## Define Router Responsiblities
 
 In your own words, define what the router does in Rails.
 
 ```md
-<!-- your answer here -->
+When a web browser makes a request, a router component checks the router file to chose which controller it will use to handle the request based on the web request and HTTP protocol.
```

Comment:
The controller action is determined by the URL and the verb, if such a route exists.